### PR TITLE
Renku UI 3.3.0

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Renku UI version 3.3.0

/deploy #persist renku-notebooks=master

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ User-Facing Changes
 * 📑 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
   accessed projects, and own datasets
   (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
+* 📑 **UI**: Show instructions on how to connect via SSH from the Session Start menu
```
Comment (lokijuhy) [inline CHANGELOG.rst:19]:
```suggestion
* 📑 **UI**: Show instructions on how to connect to RenkuLab sessions via SSH from the Session Start menu
```

**Review by lokijuhy — CHANGES_REQUESTED** [review]
I still found a _mini_ thing I wanted to change when I came back to it 😬

File: CHANGELOG.rst [hunk 2]
```diff
@@ -16,6 +16,8 @@ User-Facing Changes
 * 📑 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
```
Comment (lokijuhy) [inline CHANGELOG.rst:16]:
```suggestion
* 🧑‍💻 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -13,13 +13,16 @@ User-Facing Changes
...
-  accessed projects, and own datasets
-  (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
 * ⚙️ **Renku CLI**: Connecting to Renkulab sessions via SSH
   (`#3318 <https://github.com/SwissDataScienceCenter/renku-python/pull/3318>`_).
 * 📃 **Renku CLI**: Support for Azure Blob Storage backed datasets.
   (`#3257 <https://github.com/SwissDataScienceCenter/renku-python/pull/3257>`_).
+* 🧑‍💻 **UI**: Add a new dashboard for logged-in users, showing running sessions, last
+  accessed projects, and own datasets
+  (`#2332 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2332>`_).
+* 📑 **UI**: Show instructions on how to connect to RenkuLab sessions via SSH from the
```
Comment (aledegano) [inline CHANGELOG.rst:23]: I already asked about this but I guess I did not really understand the answer: where is this option toggled? In other words we don't need a parameter in the values to tell UI to whether to show or not these instructions?

Comment (Panaetius) [reply CHANGELOG.rst:23]: The UI gets it from notebook service which has it set here: https://github.com/SwissDataScienceCenter/renku-notebooks/blob/master/helm-chart/renku-notebooks/values.yaml#L340

Comment (aledegano) [reply CHANGELOG.rst:23]: great, thanks! The release of notebooks containing that still needs to be added here?

Comment (lorenzo-cavazzi) [reply CHANGELOG.rst:23]: Nope, that's not necessary. The UI invokes the `GET /version` API on renku-notebooks to check whether SSH is actually supported or not

**Review by lokijuhy — APPROVED** [review]

File: CHANGELOG.rst (hunk 3 above)
Comment (lokijuhy) [reply CHANGELOG.rst:23]: @aledegano You may be referring to the notebooks release PR https://github.com/SwissDataScienceCenter/renku/pull/2967, which is also on its way into this `feat/ssh-into-sessions` branch.

Comment (lorenzo-cavazzi) [reply CHANGELOG.rst:23]: (sorry haven't refreshed the page before answering)
All will be merged in the feature PR that this PR is also referencing

**Pull request merged**